bluemira/equilibria/run.py

```diff
@@ -422,7 +422,7 @@ def run_premagnetisation(self):
                 opt_conditions=self.bd_settings.opt_conditions,
                 constraints=constraints,
             )
-            result = problem.optimise(x0=max_currents, fixed_coils=False)
+            result = problem.optimise(fixed_coils=False)
```

Comment:
Yeah, it's because, before this update, the BreakdownCOP ignored x0 even if one was passed in.
So this change maintains the behaviour.

If I leave it, EUDEMO doesn't converge as well as it did before.